src/js/components/utils/useFetchTasksByProject.tsx

```diff
@@ -0,0 +1,12 @@
+import apiFetch from '~js/utils/api';
+import { addUrlParams } from '~js/utils/api';
+
+export default (
+  project: any
```

Comment:
Might as well define this, and if we go with only the `project.id` like I recommended above, then:

```suggestion
  projectId?: string
```